ansible/roles/openshift_node_config/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+# tasks file for openshift_node_config
+- name: Edit OpenShift node config
+  lineinfile:
+    dest: /etc/sysconfig/atomic-openshift-node
+    line: "{{item.key }}={{ item.value }}"
```

Comment:
the line:
```
with_items: onc_openshift_node_config
```
should probably go to:
```
with_items: "{{ onc_openshift_node_config }}"
```
to get rid of the "bare variables" warning.